tests/unit/layouts/StackLayout.spec.js

```diff
@@ -0,0 +1,88 @@
+import { expect } from 'chai';
```

Comment:
this file should be added with Stack Layout component, please remove this file

Reply:
Removed spec file